program/rust/src/tests/test_upd_price_no_fail_on_error.rs

```diff
@@ -164,11 +166,20 @@ fn test_upd_price_no_fail_on_error_no_fail_on_error() {
 }
 
 
-// Create an upd_price_no_fail_on_error instruction with the provided parameters
-fn populate_instruction(instruction_data: &mut [u8], price: i64, conf: u64, pub_slot: u64) -> () {
-    let mut cmd = load_mut::<cmd_upd_price_t>(instruction_data).unwrap();
-    cmd.ver_ = PC_VERSION;
-    cmd.cmd_ = command_t_e_cmd_upd_price_no_fail_on_error as i32;
+// Create an upd_price_no_fail_on_error or upd_price instruction with the provided parameters
+fn populate_instruction(
+    instruction_data: &mut [u8],
+    price: i64,
+    conf: u64,
+    pub_slot: u64,
+    no_fail_on_error: bool,
```

Comment:
nitpick: I think you should always define boolean flags in the positive direction, so that callers don't have to do the double negation in their head. Calling this `fail_on_error` is easier to understand.

this is a super minor issue in this instance, so feel free to merge without fixing it.